lib/gyp/generator/ninja/index.js

```diff
@@ -573,17 +583,22 @@ NinjaMain.prototype.vars = function vars() {
     cc = 'clang';
     cxx = 'clang++';
   } else if (process.platform === 'win32') {
-    ar = 'lib.exe';
-    ld = 'link.exe';
+    ar = gyp.bindings.which('lib.exe');
```

Comment:
This should be supplied by setting PATH in environment file, not by using `which`... Why is it necessary?